Modules/QuestieInit.lua

```diff
@@ -100,7 +100,7 @@ end
 
 ---Run the validator
 local function runValidator()
-    if Questie.db.global.skipValidation then
+    if Questie.db.profile.skipValidation then
```

Comment:
```suggestion
    if Questie.db.global.skipValidation then
```

And yes, we still need a checkbox for this.